blablacar-service/src/main/java/cz/fi/muni/pa165/teamred/service/facade/UserFacadeImpl.java

```diff
@@ -48,48 +49,105 @@ public Long createUser(UserCreateDTO user){
 
     @Override
     public void editUser(UserDTO userDTO) {
-        throw new NotImplementedException();
+        User user = new User();
+        user.setId(userDTO.getId());
+        user.setName(userDTO.getName());
+        user.setSurname(userDTO.getSurname());
+        user.setNickname(userDTO.getNickname());
+        userService.editUser(user);
     }
 
     @Override
     public void removeUser(Long userId) {
-        throw new NotImplementedException();
+        //redundant object creation ?
+        User userToDelete = new User();
+        userToDelete.setId(userId);
+        userService.deleteUser(userToDelete);
     }
 
-    @Override
-    public void addComment(Long userId, Long commentId) {
-        userService.addUserComment(userService.findUserById(userId),
-                commentService.findById(commentId));
-        log.info("Added Comment: " + commentService.findById(commentId).toString() +
-                "to User: " + userService.findUserById(userId).toString());
-    }
-
-    @Override
-    public void addRideAsDriver(Long userId, Long rideId) {
-        throw new NotImplementedException();
-    }
 
     @Override
     public void addRideAsPassenger(Long userId, Long rideId) {
-        throw new NotImplementedException();
+        User user = userService.findUserById(userId);
+        if (user == null){
+            log.info("No user found with id: " + userId);
+            return;
+        }
+        Ride ride = rideService.findById(rideId);
+        if (ride == null){
+            log.info("No ride found with id: " + rideId);
+            return;
+        }
+
+        userService.addUserRideAsPassenger(user, ride);
+        rideService.addPassenger(ride,user);
+        log.info("Added user: " + user +
+        " as a passenger in ride: " + ride);
     }
 
     @Override
     public void removeComment(Long userId, Long commentId) {
```

Comment:
tu chyba este removeComment z Ride + staci iba commentId ako argument, riesil by som to tak ze by som si vytiahol comment z DB a z objektu dane id kvoli konzistencii

Reply:
potom sa mi to ale viac hodi do commentFacade, pretoze vymazanie commentu suvisi viac s komentarom ako s pouzivatelom

Reply:
suhlas